Dockerfile

```diff
@@ -24,9 +24,10 @@ FROM node:${NODE_VERSION}
 
```

Comment:
Docker has recently changed. You can have a look to it at https://github.com/telefonicaid/iotagent-ul/blob/master/docker/Dockerfile

Instead of including configuration env var in Dockerfile itself, they can be added in docker-compose.yml (you can see an example at https://github.com/telefonicaid/iotagent-ul/blob/master/docker/README.md). This way, the Dockefile can be kept simple and it is easier to mantain.